src/commonMain/kotlin/baaahs/gl/glsl/GlslType.kt

```diff
@@ -136,6 +136,8 @@ sealed class GlslType constructor(
     object Sampler2D : GlslType("sampler2D")
     object Void : GlslType("void")
 
+    object SixteenFloats: GlslType("float[6]", GlslExpr("{0.,0.,0.,0.,0.,0.}"))
```

Comment:
Did this end up making it in?

Reply:
No sorry, that commit snuck into this PR. (I have prism working locally now though, just have to pack in rotation and then will have a PR ready :D).

Just force-pushed to remove that stray commit